seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/utils/CatalogUtils.java

```diff
@@ -290,7 +300,11 @@ public static CatalogTable getCatalogTable(
         }
         String catalogName = "jdbc_catalog";
         return CatalogTable.of(
-                TableIdentifier.of(catalogName, "default", "default", "default"),
+                TableIdentifier.of(
+                        catalogName,
+                        StringUtils.isBlank(databaseName) ? null : databaseName,
+                        StringUtils.isBlank(schemaName) ? null : schemaName,
+                        StringUtils.isBlank(tableName) ? "default" : tableName),
```

Comment:
Please keep the old logic. The default TablePath should be `default.default.default`.

Reply:
keep old logic, format will be somewhat unconventional.
eg：mysql-> dbname.default.tablename , ck->default.shname.tablename

Reply:
How about when `tableName` name is null, set tablepath to `default.default.default`? Otherwise keep as it is.